ui/src/album/AlbumActions.js

```diff
@@ -52,10 +79,14 @@ const AlbumActions = ({
   return (
     <TopToolbar className={className} {...sanitizeListRestProps(rest)}>
       <Button
-        onClick={handlePlay}
-        label={translate('resources.album.actions.playAll')}
+        onClick={playing ? () => dispatch(pausePlayer()) : handlePlay}
```

Comment:
Yes, this is much better. The reason I did this here is that this is the `PlayArrowIcon` and `PauseIcon` from Material UI instead of using PlayButton from `/ui/src/common`. I hope this doesn't break using /common/PlayButton in other places. But, I understood the change. Will do the needful.

Reply:
Most themes have a PLAY label next to the icon button. The spotify-ish theme for example does not. Do I conditionally render this label inside `<IconButton>` in `.../common/PlayButton` based on - 
1. Whether it is rendered on album cover 
2. Only themes that use a PLAY label
Or is there a better way to do this?

Reply:
The current PlayButton component is only used in the AlbumCover. We could make the `label` an optional attribute, so it is filled everywhere we use the button, except when adding it to the AlbumGridTile. The PlayButton should not have knowledge whether it is being added to the album cover or not.

Reply:
The IconButton currently used in PlayButton does not accept a prop called `label` (https://material-ui.com/api/icon-button/) The alternative solution is to conditionally render IconButton / Button based on the `label` prop. Replacing IconButton with Button results in some additional CSS issues. 
![image](https://user-images.githubusercontent.com/27198773/115098767-73dc9200-9f4f-11eb-8e20-6aeb54337c55.png)

Reply:
Yeah, it does not... A solution would be to pass the Button component to render, like it is done for example in the `MenuItemLink` component:
https://github.com/navidrome/navidrome/blob/b3f70538a9138bc279a451f4f358605097210d41/ui/src/layout/PersonalMenu.js#L20
Also look at how you can pass a `DatagridBody` to the `Datagrid` RA component